src/dlgprefreplaygaindlg.ui

```diff
@@ -258,7 +264,7 @@
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
-         <string>The normal average ReplayGain playback level is -14 dB. You may increase it using the inital boost, if your tracks does not require the full dynamic headroom, or reduce it if you experience that your tracks are clipping by default.</string>
+         <string>Usually, ReplayGain analysis will reduce track gain by about half, or approximately -6db. Setting an adjustment of +6db to analyzed tracks will restore ReplayGain tracks to full volume, but some tracks may clip.  Users using this setup will want to set the volume adjustment for unanalyzed tracks to 0db.  For users using a 0db adjustment to ReplayGain tracks, it's recommended to adjust unanalyzed tracks by -6db to prevent large volume changes when mixing between analyzed and unanalyzed tracks.</string>
```

Comment:
Orignal:

> The normal average ReplayGain playback level is -14 dB. You may increase it using the inital boost, if your tracks does not require the full dynamic headroom, or reduce it if you experience that your tracks are clipping by default.

New:

> Usually, ReplayGain analysis will reduce track gain by about half, or approximately -6db. Setting an adjustment of +6db to analyzed tracks will restore ReplayGain tracks to full volume, but some tracks may clip. Users using this setup will want to set the volume adjustment for unanalyzed tracks to 0db. For users using a 0db adjustment to ReplayGain tracks, it's recommended to adjust unanalyzed tracks by -6db to prevent large volume changes when mixing between analyzed and unanalyzed tracks.

This is not universal true. We may add it as an example after the original description. But such a long text was identified as too long in the first review, so we decided to put such info in the manual.